src/server/src/routes/comment.routes.ts

```diff
@@ -0,0 +1,34 @@
+import { body, param } from 'express-validator';
+import express from 'express';
+
+const router = express.Router({ mergeParams: true });
+
+import { paginate } from '../middlewares/pagination.middlewares';
+import { validate } from '../middlewares/validator.middlewares';
+import { requiredLoggedIn } from '../middlewares/auth.middlewares';
+
+import {
+    createComment,
+    getPostComments,
+} from '../controllers/comment.controllers';
+
+
+router.get('/', validate([param('post_id').isUUID()]), paginate, getPostComments);
+
+router.post(
+  '/',
+  [
+    requiredLoggedIn,
+    validate([
+      body('comment').isString(),
+      body('contentType').isIn([
+        'text/markdown',
+        'text/plain',
+      ]),
+    ]),
+  ],
+  createComment
+);
+router.get('/', paginate, getPostComments);
```

Comment:
I'm lost on what the difference is between the two endpoints

Reply:
There is no difference, I was using the post routes as a template and forgot to clean up!